by-language/java-quarkus-panache/settings.gradle

```diff
@@ -0,0 +1,11 @@
+pluginManagement {
```

Comment:
Can't you integrate this into build.gradle? is there a reason to have those separately?

Reply:
would be possible - allthough quarkus generator generates it like that - therefore I would prefer to leave it.

Reply:
Thx, then just leave it as is.